File: only C/BOJ2588.c
#include <stdio.h>

int main()
{
    int inputA, inputB;

    scanf("%d", &inputA);
    scanf("%d", &inputB);

    int _t100, _t10, _t1;
    _t100 = inputB / 100;
    _t10 = inputB / 10 % 10;
    _t1 = inputB % 10;
    // 일부는 구구절절

    printf("%d\n", inputA * _t1);
    printf("%d\n", inputA * _t10);
    printf("%d\n", inputA * _t100);

    printf("%d\n", inputA * inputB);

    return 0;
}

File: week1/BOJ2588.py
inputA = 0
inputB = 0

inputA = int(input(""))
inputB = int(input(""))

_t100 = inputB // 100
_t10 = inputB // 10 % 10
_t1 = inputB % 10

print(inputA * _t1)
print(inputA * _t10)
print(inputA * _t100)

print(inputA * inputB)